bridgeql/django/schema.py

```diff
@@ -7,6 +7,8 @@
 from django.apps import apps
 
 from bridgeql.django.settings import BridgeQLSettings
+from bridgeql.django.helpers import get_allowed_apps
+from bridgeql.django.exceptions import InvalidBridgeQLSettings
 
 
 class BridgeqlModelFields(object):
```

Comment:
General comment, should we change the class name? I am not how the name `BridgeqlModelFields` fits here. How about something like `AppMeta` or `Schema`.
cc @piyusql

Reply:
@vidya911 we talked about BridgeQLSchema earlier but **AppSchema** sounds more meaningful.

Reply:
Thanks @preyunk as always suggesting a better name :)

Reply:
Thanks @preyunk  for suggestion. I too think, AppSchema fits better will change.